src/renderer/helpers/rx/index.ts

```diff
@@ -0,0 +1 @@
+export * from './fromPromise'
```

Comment:
Pls don't create a file for just a single helper function ^. Just put `fromPromise` into a file in `src/renderer/helpers/rx.ts`, which will avoid to mess up with too many of files in our repo.